academic-services/knowledge-exploration-service/tutorial-entity-linking.md

```diff
@@ -0,0 +1,116 @@
+
+:::code language="powershell" source="linkPrivateLibraryData.ps1" id="snippet_interpret_request":::
+
+You can also link different types of entities together with the default grammar. For example, you can link author data with MAKES papers by sending Interpret requests with the following query format "**[author name] [affiliation name]**".
+
+You can also build a custom grammar to better fit your linking scenario. See [How to define a custom grammar](how-to-grammar.md) and [How to compile and test a custom grammar](how-to-grammar-build.md) for more detail
+
+## Set appropriate confidence score for linking
+
+Lastly, we'd have to determine an acceptable confidence score to link entities. Interpret API returns a log probability associated with each interpretation. This log probability can be seen as a "confidence score" for the given interpretation. Since the probability for an interpretation can range from 0 to 1, the log probability for an interpretation can be anywhere from negative infinity to 0.
+
+The interpretation log probability also included the log probability of the top matching entity. To get the pure interpretation log probability without considering the interpreted entity, we must subtract the entity log probability from the interpretation log probability.
+
+:::code language="powershell" source="linkPrivateLibraryData.ps1" id="snippet_interpret_log_probability_as_confidence_score":::
+
+The default MAKES grammar has various penalties associated with not matching paper titles directly. For example, a word matching a paper's fields of study instead of a title word will result a penalty of -1. The steepest penalty, -25 is not being able to match a word against any paper title or abstract. The sample entity linking script uses **-50** as the cut off confidence score for linking data. This allows for some penalization for entity linking, such as two title word mismatch.
+
+Depending on your linking strategy, you may have to set a different score. You may also want to design a custom grammar create a better scoring system for your scenario.
+
+## Choose MAKES entity attributes to include
+
+We start by determining which MAKES entity type we want to link our data against and what attributes can help enrich our private data. See the [MAKES Entity Schema](reference-makes-api-entity-schema.md) to explore the different types and entities and attributes available in MAKES' default index.
+
+In this tutorial, the sample private library publication records is minimal, with each record only containing a paper title and fake library URL. The goal of the tutorial is to augment the minimal library paper records with additional metadata from MAKES, enabling a robust semantic/keyword search experience.
+
```

Comment:
You should include a new section after "Prerequisities" that describes the scenario covered by the tutorial and includes this (+additional) information. Maybe something like...

## Download samples and setup working directory

This tutorial illustrates how a fictional library could go about linking their existing, minimal library records to the much more expansive MAG/MAKES metadata, allowing them to enhance their records and create a powerful library search website.

To get started with entity linking, create a working directory on you local filesystem and download the [sample publication records](samplePrivateLibraryData.json) and the [sample entity linking script](linkPrivateLibraryData.ps1) to it.

As mentioned, the sample library publication records contain minimal information:
* Publication title
* URL for the publication on the library's website

Once the working directory has been setup, our next step is to determine the entity linking strategy.